go.mod

```diff
@@ -204,3 +204,5 @@ require (
 replace github.com/ethereum/go-ethereum => github.com/iotexproject/go-ethereum v0.4.2
 
 replace golang.org/x/xerrors => golang.org/x/xerrors v0.0.0-20190212162355-a5947ffaace3
+
+replace github.com/iotexproject/go-p2p => /Users/chenchen/dev/go-p2p
```

Comment:
create a PR for `go-p2p` changes

Reply:
sure, it's here: https://github.com/iotexproject/go-p2p/pull/36